stl/src/ppltasks.cpp

```diff
@@ -235,7 +235,7 @@ namespace Concurrency {
             } else {
                 ComCallData callData;
                 ZeroMemory(&callData, sizeof(callData));
-                callData.pUserDefined = reinterpret_cast<void*>(&_Func);
+                callData.pUserDefined = &_Func;

```

Comment:
Note for the record, no changes requested: At first, I thought this might be taking advantage of non-Standard behavior regarding casting function pointers to object pointers (as mentioned in a previous comment). However, `_Func` is a `std::function`, as `ppltasks.h` declares:
```cpp
typedef ::std::function<void __cdecl ()> _CallbackFunction;
```
Therefore, removing the cast here is absolutely the right thing to do.